bokehjs/src/lib/document/document.ts

```diff
@@ -108,6 +110,8 @@ export class Document implements Equatable {
   protected _interactive_plot: Model | null
   protected _interactive_finalize: (() => void) | null
   protected _recompute_timeout: number
+  protected _connection_el: HTMLElement | null = null
```

Comment:
Not following this completely. This is there by default? I'm not super convinced we should build this into the Document by default.